extensions/ql-vscode/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
This makes me a bit uncomfortable as it could potentially introduce breaking changes (from my understanding). Is it possible to avoid it and add the dependency that is causing the trouble explicitly?

Reply:
Unfortunately, this is the recommended fix by Storybook (see https://github.com/storybookjs/storybook/issues/18298). I'm not sure we can avoid adding the dependency since it's one of the Storybook packages which causes this to be necessary.

Reply:
Thanks for the link! It mentions that this will be fixed with storybook v7. Is it worth using an alpha version of that rather than 6? The reason I'm asking is because the current change potentially affects the actual extension ("prod") but using an alpha version of the storybook only potentially makes things a bit unstable for dev. 

I'm saying that without any idea about storybook versions and how mature the alpha is though!

Reply:
The PR that is referenced in the issue has also been backported to Storybook 6, but it does not actually fix the problem. The workaround which was implemented is to automatically set this parameter in the `.npmrc` file (which is how it got added here). It doesn't seem like they have a real fix yet.

Reply:
Ah 💩 okay. Thanks for the links and explanations!

Shall we add a comment to this file to explain why we're setting this flag?